submissions/Kostiantyn_Rakhmanov/hooli-style-popup/index.html

```diff
@@ -0,0 +1,154 @@
+                            <div class="products__more">
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/chat.png" alt="Chat">
+                                        <p class="product__item-name">Chat</p>
+                                    </a>
+                                </div>
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/classroom.png" alt="Classroom">
+                                        <p class="product__item-name">Classroom</p>
+                                    </a>
+                                </div>
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/collection.png" alt="Collection">
+                                        <p class="product__item-name">Collection</p>
+                                    </a>
+                                </div>
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/mars.png" alt="Mars">
+                                        <p class="product__item-name">Mars</p>
+                                    </a>
+                                </div>
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/blogs.png" alt="Blogs">
+                                        <p class="product__item-name">Blogs</p>
+                                    </a>
+                                </div>
+                                <div class="product__item">
+                                    <a class="product__item-link" href="#">
+                                        <img class="product__item-img" src="./img/promo.png" alt="Promos">
+                                        <p class="product__item-name">Promos</p>
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="menu__item">
+                        <a class="menu__link" href="#"><img class="notification" src="./img/bell.png"
+                                alt="notification"></a>
+                    </li>
+                </ul>
+            </nav>
+            <img class="avatar" src="./img/favicon.png" alt="avatar">
```

Comment:
What is this image for?

Reply:
Hi! 
Main idea of this image is - default avatar of user. For example, Like in gmail account.
Is it better to remove it?
p.s. Thanks for review! Will do changes ASAP.

Reply:
So if it's a part of page navigation it must be inside the `nav` tag and be a link or button and be focusable)